katsdpcontroller/generator.py

```diff
@@ -1737,7 +1740,7 @@ def _get_targets(config, capture_block_id, name, telstate, min_time):
         else:
             logger.info('Skipping target %s: observed for %.1f seconds, threshold is %.1f',
                         target.name, obs_time, min_time)
-    return targets
+    return targets, data_set
```

Comment:
I'll be making some continuum imager changes that will require 'band awareness' in the not-too-distant future so this extra plumbing will come in handy there as well.